### PR TITLE
Refactoring pt2

### PR DESCRIPTION
**Pull request by JCBeldo**
This cleans up some code lines and deletes commented out code.

### CODE REVIEW

File: lib/board.rb [hunk 1]
```diff
@@ -40,84 +40,91 @@ def valid_placement?(ship, ship_coordinates)
...
-      "D #{@cells["D1"].render(true)} #{@cells["D2"].render(true)} #{@cells["D3"].render(true)} #{@cells["D4"].render(true)}"
-      else
-      "  1 2 3 4 \n" +
-      "A #{@cells["A1"].render} #{@cells["A2"].render} #{@cells["A3"].render} #{@cells["A4"].render}\n" +
-      "B #{@cells["B1"].render} #{@cells["B2"].render} #{@cells["B3"].render} #{@cells["B4"].render}\n" +
-      "C #{@cells["C1"].render} #{@cells["C2"].render} #{@cells["C3"].render} #{@cells["C4"].render}\n" +
-      "D #{@cells["D1"].render} #{@cells["D2"].render} #{@cells["D3"].render} #{@cells["D4"].render}"
-    
-     end
+      rendered_true
```
Comment (brenicillin) [inline lib/board.rb:51]: great job shortening and refactoring into helper methods!

File: lib/game.rb [hunk 2]
```diff
@@ -85,29 +86,30 @@ def player_setup
   end
   
   def display_boards
-    puts " "
-    puts "==========COMPUTER BOARD=========="
-    puts @cpu_board.render
-    puts "===========PLAYER BOARD==========="
-    puts @player_board.render(true) #ask about writing this test
-    puts " "
+    print "\n"
```
Comment (brenicillin) [inline lib/game.rb:89]: great use of line breaks to improve readability

File: lib/ship.rb [hunk 3]
```diff
@@ -9,11 +9,7 @@ def initialize(name, length)
   end
 
   def sunk?
-    if @health == 0
-      true
-    else
-      false
-    end
+    @health.zero?
```
Comment (brenicillin) [inline lib/ship.rb:12]: more great refctoring, love that this is one line now

**Review by brenicillin — APPROVED** [review]
great refactoring!

**Comment by JCBeldo** [comment]
I am electric?

**Comment by brenicillin** [comment]
you are electric

**Pull request merged**